group_vars/sn06.yml

```diff
@@ -963,6 +974,18 @@ galaxy_jobconf:
         dependency_resolution: remote
         rewrite_parameters: "true"
         persistence_directory: /data/share/persisted_data
+    - id: pulsar_mira_tpv
```

Comment:
Are you sure this needs to be here? Can this be also in the tpv configuration? Maybe a question for @Slugger70 ?

Reply:
Yes. The destination needs to be in the job conf the same as in the TPV destinations.